### DISCUSSION TITLE
A "Files" M2M Field With Image Previews

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#17137] [Feature Requests]
The icon for the "Files" M2M field is rather deceptive. It looks like what I've going to get is a nice grid where I can visually see the images I've added to the field.

<img width="163" alt="image" src="https://user-images.githubusercontent.com/26592486/212176824-2a26c6bd-63af-4ef0-95cb-b3d0fcc36b27.png">

That's not at all what I get. The files are displayed in a simple list. This would be great if the files were documents or audio or something, but for images it would be much more helpful to be able to see some sort of preview for each one.

<img width="796" alt="image" src="https://user-images.githubusercontent.com/26592486/212177423-2f673053-e76e-433d-9bae-2206d34d4804.png">

It would be helpful to even just be able to add a little circle image beside each image name so there is some small visual preview. I can do this for items in the collection list, so why can't I do it here?

**Comment by paescuj** [comment]
> The icon for the "Files" M2M field is rather deceptive.

Hmm, in my opinion it is quite fitting if you compare it with icon from the single file / image field next to it.

> This would be great if the files were documents or audio or something, but for images it would be much more helpful to be able to see some sort of preview for each one.

This is what it's actually supposed to do. Here's what it looks like in my case:

![Screenshot 2023-01-13 at 16-28-01 Creating Item in Test](https://user-images.githubusercontent.com/5363448/212356970-fc834d1a-89a6-4384-810b-608f1a16f253.png)

- Are you on the latest version of Directus?
- Did you adjust the configuration of the field? (In the screenshot above, the field is set-up with the default settings)
- Are all the referenced files images?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
